### PR TITLE
Generate a token for the callback scheme

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

[VIH-5482](https://tools.hmcts.net/jira/browse/VIH-5482)

### Change description ###

Fix the setup for callback bearer token

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs [hunk 1]
```diff
@@ -30,12 +30,12 @@ public abstract class BaseServiceTokenHandler : DelegatingHandler
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
             CancellationToken cancellationToken)
         {
-            var token = GetServiceToServiceToeken();
+            var token = GetServiceToServiceToken();
```
Comment (rdbartram) [inline VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs]: this should not have been merged originally. ridiculous!

Comment (npoxon) [reply VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs]: ToeKen. Seriously? GET. A. GRIP.

File: VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs [hunk 2]
```diff
@@ -30,12 +30,12 @@ public abstract class BaseServiceTokenHandler : DelegatingHandler
...
             request.Headers.Add("Authorization", $"Bearer {token}");
             return await base.SendAsync(request, cancellationToken);
         }
         
-        protected string GetServiceToServiceToeken()
+        protected string GetServiceToServiceToken()
```
Comment (rdbartram) [inline VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs]: here the same!
----
this should not have been merged originally. ridiculous!

Comment (shaed-parkar) [reply VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs]: okay done now

**Pull request merged**